be/src/http/action/download_action.cpp

```diff
@@ -114,6 +126,14 @@ void DownloadAction::handle_error_log(HttpRequest* req, const std::string& file_
 }
 
 void DownloadAction::handle(HttpRequest* req) {
+    if (_is_async) {
```

Comment:
why is this asynchronous? Could you add some comments please?

Reply:
In my opinion, download job for single replica maybe a relatively heavy task, and it seems that dispatching their process logic to a standalone worker pool maybe more reasonable, and let free the IO threads  to some extent.

Do u think it is reasonable to do so? If so, I'll make some comments to clarify it, otherwise I'll modify the logic.